com.unity.mobile.notifications/Runtime/Android/AndroidNotification.cs

```diff
@@ -57,284 +57,158 @@ public struct AndroidNotification
         /// Notification title.
         /// Set the first line of text in the notification.
         /// </summary>
-        public string Title
-        {
-            get { return title; }
-            set { title = value; }
-        }
+        public string Title { get; set; }
```

Comment:
the package is compatible with 2018.4, does this code work if you set .NET 3.5?

Or there's a requirement that .NET 4.x must be used?

Reply:
Yes. Auto-Implemented Properties is supported since C# 3.0, https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties.

I guess you're referring to Expression body definitions which is supported since C# 7.0 https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties#expression-body-definitions ?